MAUI/DataGrid/columns.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: Columns in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Columns support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control, its elements and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Columns in MAUI DataGrid (SfDataGrid)
+
+The [SfDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html) allows to create and add columns in the following two ways,
+
+* Automatically generating columns
+* Manually defining columns
+
+## Automatic columns generation
+
+The SfDataGrid creates columns automatically based on the bindable property [SfDataGrid.AutoGenerateColumnsMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_AutoGenerateColumnsMode). The columns are generated based on the type of individual properties in the underlying collection that is set as ItemsSource.
+
+Below table shows the column type created for the respective data types. For remaining types, [DataGridTextColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridTextColumn.html) will be created.

```

Comment:
The table below shows the column type created for the respective data types. For the remaining data types, [DataGridTextColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridTextColumn.html) will be created.